src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiersTest.java

```diff
@@ -69,21 +69,23 @@ public void should_not_verify_invalid_filenames_successfully() throws Exception
 
     @Test
     public void should_verify_valid_zip_successfully() throws Exception {
-        byte[] test1Bytes = toByteArray(getResource("signature/1_24-06-2018-00-00-00.test.zip"));
+        String zipFileName = "1_24-06-2018-00-00-00.test.zip";
+        String container = "signature";
+        byte[] test1Bytes = toByteArray(getResource(container + "/" + zipFileName));
```

Comment:
wouldn't it be clearer if `signature` remained to be a subfolder in resources for tests and something like `some_container` was used as a container parameter?

Reply:
No it wouldn't. That's why I did not do it that way :-)